docs/Makefile

```diff
@@ -1,232 +1,20 @@
-# Makefile for Sphinx documentation
+# Minimal makefile for Sphinx documentation
```

Comment:
Sphinx creates a Makefile when you initialize a Sphinx project. The Makefile we had is from running that a long time ago. So I updated it.